README.md

```diff
@@ -2,3 +2,13 @@
 [![Build Status](https://travis-ci.org/DFE-Digital/login.dfe.help.svg?branch=master)](https://travis-ci.org/DFE-Digital/login.dfe.help)
 [![tested with jest](https://img.shields.io/badge/tested_with-jest-99424f.svg)](https://github.com/facebook/jest)
 
+#Running Compose and docker
+ make sure you have docker desktop runnning
+ select the compose file you want to run 
+ right click the file select compose up
+ or alternately run default compose command in the terminal window
+ docker compose up
+ and to remove completely right click the file you composed up and select
+ compose down
+ or alternatively in the ternimal window
+ docker compose down
```

Comment:
add some description for docker debug compose if some one like to debug they have to compose up debug file